src-docs/src/views/combo_box/combo_box_example.js

```diff
@@ -252,7 +252,11 @@ export const ComboBoxExample = {
           that the user needs to be able to search them, the user needs to be
           able to select multiple options, and/or the user should have the
           ability to specify a custom value in addition to selecting from a
-          predetermined list.
+          predetermined list. See{' '}
+          <EuiLink href="https://github.com/elastic/eui/discussions/7049">
```

Comment:
Any thoughts on making this a link that opens in a new tab (since it links to GH rather than to our internal docs)? I'd be good either way, just wanted to get your thoughts
```suggestion
          <EuiLink href="https://github.com/elastic/eui/discussions/7049" target="_blank">
```

Reply:
That would be helpful. I've updated all four links to open in a separate tab
https://github.com/elastic/eui/pull/7053/commits/16edd004d426c6a7166f2db7d8635f798828285b